### PR TITLE
[2550] Dont show GetStarted for empty commit

### LINKED ISSUE #2550: Get-started is shown for an empty past commit
Steps to reproduce:
1. From the UI, create a repository.
2. Upload a file.
3. From the uncommitted tab, commit the change.
4. From the Objects tab, click the "branch: main" drop down.
5. Click the arrow on the right.
6. Select the first commit with the "Repository created" message.

Result: the "get started" screen appears.
Expected: screen should be empty, because this is a past commit.

image

### PR DESCRIPTION
**Pull request by tshradheya**
Fixes #2550

Only showing the GetStarted Page when the user is on main branch and branch is empty.

Selecting refs would show results even if empty

### CODE REVIEW

**Comment by tshradheya** [comment]
@talSofer is there a way for me to update the label to pass all checks?

File: webui/src/lib/components/repository/tree.jsx [hunk 1]
```diff
@@ -288,7 +288,7 @@ const GetStarted = ({ onUpload }) => {
 export const Tree = ({ repo, reference, results, after, onPaginate, nextPage, onUpload, onDelete, showActions = false, path = "" }) => {
 
     let body;
-    if (results.length === 0 && path === "") {
+    if (results.length === 0 && path === "" && reference.type === "branch") {
```
Comment (nopcoder) [inline webui/src/lib/components/repository/tree.jsx:291]: We can use `RefTypeBranch` here.

**Review by nopcoder — CHANGES_REQUESTED** [review]

**Comment by nopcoder** [comment]
@tshradheya thanks for the contribution, I've updated the required label and added a small comment to your change.

**Review by nopcoder — APPROVED** [review]
LGTM!

**Pull request merged**